addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -931,15 +945,18 @@ function formatTables($editable) {
             }
         }
     }
-    // Tables don't properly inherit alignments from their ancestors in Outlook.
+    // Tables don't properly inherit certain styles from their ancestors in Outlook.
```

Comment:
This change is quite funny because it's basically saying: "oops, looks like there are yet other things that are broken in Outlook" :p